lib/cli/devMode.js

```diff
@@ -37,12 +37,7 @@ async function watchLiquidTest(firmId, handle, testName, html_render) {
     lastUpdates[filename] = fileStats.mtimeMs;
 
     // Run test
-    await liquidTestRunner.runTestsWithOutput(
-      firmId,
-      handle,
-      testName,
-      html_render
-    );
+    await liquidTestRunner.runTestsWithOutput(firmId, handle, testName, mode);
```

Comment:
`runTestsWithOutput` accepts `htmlInput` and `htmlOutput` as parameters, not `mode`.
Actually I think like more the  idea of passing `mode` and doing the mode check outside the function ?
Also, we could rename `mode` to `htmlMode` in the parameters

Reply:
I think this is something I hadn't gotten to yet cause it was in draft

Reply:
I still think this is not correct: `mode` refers to `previewOnly` parameter? I would at least rename it to make it more clear.
So when we use `--html` in `development-mode` we want only the HTML (from input?) and not the output in the terminal ?
I'm getting this error when I add the `--html` flag

![Screenshot from 2023-09-12 12-30-53](https://github.com/silverfin/silverfin-cli/assets/99397124/756af8ec-8925-4704-b4a4-00b4fb21f698)

Also, if we use `--preview-only` we must use at least one of `--html-input` or `--html-preview`. If non of them are added which preview will we take?

I get exactly the same error as before (and I think that's the issue in `development-mode`) while running:
`silverfin run-test -h handle --preview-only`

Reply:
This should be working now, `development-mode` will render only input-mode of the template. I think the auto-rendering of the input/preview HTMLs will be cleaner to integrate in the extension for now or already have the option by using `run-test` manually. 

Cause when using the CLI `development-mode` for rendering the different HTMLs on the liquid tests, it really spams you with tabs so I'd also want to fix the opening of the page in a new tab every time when integrating this completely in this command (so maybe better to take that into a separate PR)